package.json

```
@@ -10,6 +10,11 @@
   },
   "dependencies": {
     "@apollo/client": "3.7.0",
+    "@emotion/react": "^11.10.4",
+    "@emotion/server": "^11.10.0",
+    "@mantine/core": "^5.6.1",
+    "@mantine/hooks": "^5.6.1",
+    "@mantine/next": "^5.6.1",
```

Could you, please, remove carets `^` from the package versions?
```suggestion
    "@emotion/react": "11.10.4",
    "@emotion/server": "11.10.0",
    "@mantine/core": "5.6.1",
    "@mantine/hooks": "5.6.1",
    "@mantine/next": "5.6.1",
```

---

Also, merge current `development` into your branch and solve conflicts for `package-lock.json`.

---

Hey I fixed the issues, check it out. I think that prettier configuration is not right, eslint was not happy with the code formatting so I had to manually format the code in accordance to eslint, but that issue is out of the scope for this pull request. So, please merge it.